MauldinEconomics/mautic
# Permissions

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? |  x
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Add export leads permissions. 
Add report  export permissions.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  
2. 

#### Steps to test this PR:
1. Create a new user add export permissions
2. Login with the new user 
3. Open Contacts page check if there is export button
3. Open Reports page check if there is export button

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
